modules/aws_ebs_csi/output.tf

```diff
@@ -0,0 +1,3 @@
+output "ebs_csi_account" {
```

Comment:
File name should be outputs.tf

Reply:
file name updated, interesting that TF must accept both name variants as I had copied output.tf from within the aws_autoscaling module folder.

Reply:
terraform really does not care what you call the files as long as they end in .tf It's more of a coding standard.